draft-ietf-spice-sd-cwt.md

```diff
@@ -905,15 +892,32 @@ This section provides a non-exhaustive list of topics to be considered when deve
    5. Does the credential type require high precision time or other claims that have sufficient entropy such that they can serve as a unique fingerprint for a specific subject?
    6. Does the credential type contain Personally Identifiable Information (PII), or other sensitive information that might have value in a market?
 
-3. Verifier questions:
+3. Holder question:
+
+Is there a way to prohibit extortion of over-identification or disclosure misuse via a Verifier by a Holder being bullied into disclosure towards Verifiers?
```

Comment:
```suggestion

0. What steps has the Holder taken to improve their operation security regarding presenting credentials to verifiers?
1. How can the Holder be convinced the Verifier that received presentations is legitimate?
2. How can the Holder be convinced the Verifier will not share, sell, leak, or otherwise disclose the Holder's presentations or Issuer or Holder signed material?



Is there a way to prohibit extortion of over-identification or disclosure misuse via a Verifier by a Holder being bullied into disclosure towards Verifiers?
```